### DISCUSSION TITLE
Unable to build code locally - .Net 10 SDK not found

### DISCUSSION THREAD

**Discussion by ks1990cn** [dotnet/aspnetcore#59307] [Q&A] [answered]
Hi,

Can anyone help me with repo setup?

I have gone through https://github.com/ks1990cn/aspnetcore/blob/main/docs/BuildFromSource.md.

Step I did - 
- I ran ./restore in aspnetcore (restoring runtime .net 10)
- I ran ./build.cmd for specific solution
- I opened visual studio
- I still get error that sdk is not present.

<img width="782" alt="image" src="https://github.com/user-attachments/assets/412f48e3-b3eb-4ab7-b7cd-0be7a6552fd2">

**Comment by martincostello** [comment, marked as answer]
Did you open Visual Studio using `./startvs.cmd` from the folder with the solution you wanted to open it from?

**Reply by ks1990cn** [reply]
Yes

**Reply by ks1990cn** [reply]
I tried again with another project - Middleware. Getting below error:

<img width="908" alt="image" src="https://github.com/user-attachments/assets/34ee22f2-bd0d-47c8-a6bb-f30d873722c0">

**Reply by ks1990cn** [reply]
I think, this is another issue can be put in another discussion. Thank you so much
